packages/common/package.json

```diff
@@ -11,8 +11,7 @@
         "@types/js-yaml": "^4.0.9",
         "@types/pegjs": "^0.10.3",
         "@types/sanitize-html": "^2.11.0",
-        "@types/uuid": "^10.0.0",
-        "typescript-json-schema": "^0.54.0"
```

Comment:
unused?

Reply:
`typescript-json-schema` yeah. and this module was resolving older typescript 🤷